Dockerfile

```diff
@@ -14,11 +14,13 @@ ENV WEB_SERVER_UWSGI_ASYNC_CORES 10
 ENV HTTP_KEEP_ALIVE 2
 ENV GUNICORN_CMD_ARGS -c gunicorn_config.py
 
-COPY Pipfile Pipfile
-COPY Pipfile.lock Pipfile.lock
+COPY pyproject.toml pyproject.toml
+COPY poetry.lock poetry.lock 
 
 RUN groupadd -r appuser && useradd -r -g appuser -u 9000 appuser && chown -R appuser:appuser .
-RUN pip install pipenv==2023.8.22 && pipenv install --deploy --system && make build
+RUN pip install "poetry==1.7.1"
```

Comment:
To prevent introducing additional image layers with the separate `RUN` statements here, could these commands be grouped sequentially like in a similar way that the pipenv commands were? So something like:
```
RUN pip install "poetry==1.7.1" && \
    poetry config virtualenvs.create false && \
    poetry install --only main && \
    make build
```
This might slightly reduce the image size also? No worries if this has already been tried though!

Reply:
Installation with a single `Run` statement has been added 👍